src/reducers/browse.js

```diff
@@ -25,6 +25,20 @@ function browse(state = initialBrowseState, action) {
     case actionTypes.GET_SCHOOLS_FAILURE:
       return state.set('isFetching', false)
         .set('schools', initialBrowseState.get('schools'));
+    case actionTypes.FETCH_SUBJECTS_REQUEST:
+      return loop(
+        state.set('isFetching', true),
+        Cmd.run(fetchSubjects, {
+          successActionCreator: getSubjectsSuccess,
```

Comment:
I think you meant `fetchSubjectsSuccess`

Reply:
Oh yes I'll change the gets to fetches for the subject actions